src/lib/ndpi_content_match.c.inc

```diff
@@ -717,6 +717,31 @@ static ndpi_network host_protocol_list[] = {
   { 0x3353EED8 /* 51.83.238.216/30 */, 30, NDPI_PROTOCOL_ANYDESK },
   { 0x3353EEDC /* 51.83.238.220/31 */, 31, NDPI_PROTOCOL_ANYDESK },
 
+  /*
+   * Edgecast CDN
+   * Uses captcha to prevent web scraping.
```

Comment:
A captcha to protect a list of 20 addresses, last updated 8 years ago.... :(

Reply:
agreed